cmd/zpool/zpool.d/media

```diff
@@ -18,6 +18,13 @@ if [ -b "$VDEV_UPATH" ]; then
 	if [ "$val" = "1" ]; then
 		MEDIA="hdd"
 	fi
+
+	vpd_pg83="/sys/block/$device/device/vpd_pg83"
+	if [ -f "$vpd_pg83" ]; then
+		if grep -q --binary "iqn." "$vpd_pg83"; then
+			MEDIA="iscsi"
```

Comment:
Since iscsi is the transport protocol, would it make sense to append `iscsi` to the disk type, like `iscsi-hdd` or `iscsi-ssd`? (I have not used iscsi so this may be a dumb question).

Reply:
I don't think the iSCSI protocol exposes the backend device type. But I'll try an experiment with both backends to see if we can see it.  If so then a hybrid might make sense.